addons/website/static/src/js/editor/widget_link.js

```diff
@@ -26,43 +28,77 @@ weWidgets.LinkTools.include({
     },
     /**
      * Allows the URL input to propose existing website pages.
-     *
+     * Allows the URL input to be filled from document upload.
+     * Automatically creates a callable link if selected content is a phone number.
+     * 
```

Comment:
Whitespace, I can send you my eslint or there is one on the odoo GH wiki I think.
Can't miss it on Atom :D 
![image](https://user-images.githubusercontent.com/30048408/116274804-4a0f4080-a783-11eb-9b95-4eb974cd2dc0.png)